pkg/kubelet/kubelet.go

```diff
@@ -557,11 +558,78 @@ func (kl *Kubelet) Run(updates <-chan PodUpdate) {
 	kl.syncLoop(updates, kl)
 }
 
+func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
+	node := &api.Node{
+		ObjectMeta: api.ObjectMeta{Name: kl.hostname},
+		Status: api.NodeStatus{
+			Capacity: api.ResourceList{
+				api.ResourceCPU:    *resource.NewMilliQuantity(1000, resource.DecimalSI),
```

Comment:
I think this code was in NodeController so there would be some capacity in NodeStatus between the time the node is created in the master, and the first time the node reports its status.